src/fight.cpp

```diff
@@ -5551,10 +5551,17 @@ void chkdmg(struct veh_data * veh)
     struct char_data *i, *next;
     struct obj_data *obj, *nextobj;
     int damage_rating, damage_tn;
-
+    extern int max_npc_vehicle_lootwreck_time;
+      
     // Remove any vehicle brains, we don't want them thrown into the street.
     remove_vehicle_brain(veh);
-
+    
+      if (veh->owner <= 0) {
+        veh->locked = TRUE; // Keeps players from messing around with it.
```

Comment:
This means they can't tow it, is that the intent?

Reply:
Can't tow, can't pick it up, and can't repair it. I don't want people messing around with something that will despawn.

Reply:
This means that all NPC vehicles, once wrecked, can't be towed, repaired, joyridden, etc. Any idea if people are currently doing that stuff?

Reply:
People are driving forklifts and such, and might feel obligated to repair them, but with the decay mechanic they should just respawn, shouldn't they?

People do occasionally do that with the lone star vehicles and a few other things, but the novelty wears off quickly. I'd be more inclined to make a more complicated system (based on what I've written in the issue tracker on the github),  but that requires way more input then I currently have.